pkg/codegen/internal/test/testdata/simple-enum-schema/python/pulumi_plant/tree/v1/_enums.py

```diff
@@ -27,8 +27,11 @@ class RubberTreeVariety(str, Enum):
     types of rubber trees
     """
     BURGUNDY = "Burgundy"
+    "A burgundy rubber tree."
```

Comment:
Here it is :)

Reply:
I wonder if you need `"""` to indicate it's a docstring to the tooling. Otherwise LGTM!

Reply:
Right, this is my fix :)

Reply:
ah good point!